index.html

```diff
@@ -47,79 +51,116 @@ <h2>Let's Connect</h2>
         </section>
         <section class="container">
             <div class="card">
-                <img class="first-image" src="/images/1.png" alt="">
+                <div id="first">
```

Comment:
Good job working on the form validation 👍.  but there is still room for improvement 😃 . Kindly make sure you stuck to the design as much as possible (font, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)

- [ ] Kindly make sure your section `<section class="container">` his `background` `#E5E5E5` macht with the [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Kindly, make sure the `about` section doesn't have the `boder` as we can see on the [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/229522557-62b8ea3a-41b2-4231-bd04-5750629693f5.JPG)| |![Capture0](https://user-images.githubusercontent.com/57408419/229523516-b3d29e37-7f2a-493c-8301-75fd31e67106.JPG)| |

- [ ] Also, make sure your `home page` has a `border-bottom-left-radius` as we can see on the [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![home0](https://user-images.githubusercontent.com/57408419/229527503-4b22ca91-01d3-4b9d-aaa8-367364a977bf.JPG)| |![home](https://user-images.githubusercontent.com/57408419/229527420-16ed2ce9-582b-4737-b293-ea63451fab31.JPG)| |